Jenkinsfile

```diff
@@ -132,7 +134,8 @@ builders['msvc2015'] = {node('windows'){
             withEnv(["PATH+QT=${env.QT}\\Tools\\${version}\\bin;${env.QT}\\5.7\\${version}\\bin",
                      "PATH+CMAKE=${env.CMAKE}\\bin",
                      "CMAKE_PREFIX_PATH+QT=${env.QT}\\5.7\\${version}",
-                     "GOOGLETEST_DIR=${tool 'googletest'}"]){
+                     "GOOGLETEST_DIR=${tool 'googletest'}",
+                     "EMBED_PYTHON_ARCHIVE_PATH=${tool 'embed-python-32'}"]){
```

Comment:
Those env variables aren't automatically propagated as CMake variables - you can see that `GOOGLETEST_DIR` is used below as cmake execution parameter (I don't even know why it is set as env variable instead of being directly provided there).